R/drive_cp.R

```diff
@@ -57,9 +66,11 @@ drive_cp <- function(file, path = NULL, name = NULL, verbose = TRUE) {
 
   name <- name %||% glue("Copy of {file$name}")
 
-  params <- list(
+  dots <- list(...)
+  dots$fields <- dots$fields %||% "*"
+  params <- c(
     fileId = file$id,
-    fields = "*"
+    dots
   )
```

Comment:
This is how I propose we deal with the possibility that user passed `fields` in `...`. I also now think we should default to `fields = "*"` everywhere and see if there's ever real world justification for working any harder at this.